dns_server.go

```diff
@@ -38,7 +38,8 @@ type DNSServer struct {
 	Interval              int            `json:"interval,omitempty"`
 	MtuMeasurements       int            `json:"mtuMeasurements,omitempty"`
 	NetworkMeasurements   int            `json:"networkMeasurements,omitempty"`
-	NumPathTraces         int            `json:"numPathTraces,omitempty"`
+	NumPathTraces         int            `json:"numPathTraces"`
```

Comment:
What does the omitempty need to leave for on this? Can it be a valid 0?

Reply:
https://developer.thousandeyes.com/v6/tests/#/test_metadata looks like its 1..10 everywhere?

Reply:
yea, so I guess it doesnt really matter either way since it cant ever be empty... want me to remove it  ?

Reply:
It’s fine, just leave it will merge it can’t be empty and has a default so it’s fine for now